adp-aso-helm-library/templates/_flexible-servers-db.yaml

```diff
@@ -0,0 +1,26 @@
+{{- define "adp-aso-helm-library.flexible-servers-db.tpl" -}}
+{{- $requiredMsg := include "adp-aso-helm-library.default-check-required-msg" . -}}
+apiVersion: dbforpostgresql.azure.com/v1api20220120preview
+kind: FlexibleServersDatabase
+metadata:
+  {{ $serverName := required (printf $requiredMsg "postgresServerName") .Values.postgresServerName }}
+  {{ $dbName := required (printf $requiredMsg "postgres.db.name") .Values.postgres.db.name }}
+  name: {{ printf "%s-%s" $serverName $dbName }}
+  namespace: {{ .Values.namespace }}
+  {{ with .Values.asoAnnotations }}
+  annotations:
+    {{ toYaml . | nindent 4 }}
+  {{ end }}
+spec:
+  azureName: {{ .Values.postgres.db.name }}
+  charset: {{ .Values.postgres.db.charset }}
+  collation: {{ .Values.postgres.db.collation }}
+  owner:m
+    armId: {{ printf "/subscriptions/%s/resourceGroups/%s/providers/Microsoft.DBforPostgreSQL/flexibleServers/%s" .Values.subscriptionId .Values.postgresResourceGroupName .Values.postgresServerName }}
+  tags:
+  {{ include "adp-aso-helm-library.tags" . | nindent 4}}
```

Comment:
Tag is not a property on database. Can you please remove it.
https://azure.github.io/azure-service-operator/reference/dbforpostgresql/v1api20210601/#dbforpostgresql.azure.com/v1api20210601.FlexibleServers_Database_Spec

Reply:
done